### PR TITLE
magento2-login-as-customer/issues/140: Incorrect position of Login as Customer Log

### PR DESCRIPTION
**Pull request by engcom-Foxtrot**
### Fixed Issues (if relevant)

1. Fixes https://github.com/magento/magento2/issues/29111: Incorrect position of Login as Customer Log

### Manual testing scenarios (*)

1. See https://github.com/magento/magento2/issues/29111

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by engcom-Delta** [comment]
:heavy_check_mark: QA passed
**Result:**
image

File: app/code/Magento/LoginAsCustomerLog/etc/adminhtml/menu.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
         <add id="Magento_LoginAsCustomerLog::login_log"
              title="Login As Customer Log"
              module="Magento_LoginAsCustomerLog"
-             parent="Magento_Customer::customer"
+             parent="Magento_AsynchronousOperations::system_magento_logging"
```
Comment (naydav) [inline app/code/Magento/LoginAsCustomerLog/etc/adminhtml/menu.xml:14]: It's not good to introduce soft dependency `Magento_AsynchronousOperations`
Please duplicate creating of "Action log" section in this module

Comment (engcom-Foxtrot) [reply app/code/Magento/LoginAsCustomerLog/etc/adminhtml/menu.xml]: @naydav duplicating creates second menu item 
Screenshot from 2020-05-12 13-20-55

Comment (engcom-Foxtrot) [reply app/code/Magento/LoginAsCustomerLog/etc/adminhtml/menu.xml]: @naydav duplicate "Action log" section in this module without id changing causes fatal exception, while changing id causes menu items duplication 
Screenshot from 2020-05-12 13-20-55

**Pull request closed without merging**